### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by cocovsack**
Modify homepage navigation bar and layout

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,14 +15,73 @@
...
 }
+
+//immediately after window loads, call function
+window.onload = function() {
+  var text_elements = document.getElementsByClassName('txt-rotate');
+  for (var i = 0; i < text_elements.length; i++) {
+    var toRotate = text_elements[i].getAttribute('data-rotate');
+    var period = text_elements[i].getAttribute('data-period');
+    if (toRotate != null) {
+      new TxtRotate(text_elements[i], JSON.parse(toRotate), period);
```
Comment (liguanghua) [inline portfolio/src/main/webapp/script.js:37]: Is "new" needed?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,79 @@
...
+
+.container{
+    display: inline;
+}
+.titleImage {
+    max-height: 500px;
+    max-width: 500px;
+    object-fit: cover;
+    right: 0px;
+    bottom: 0px;
```
Comment (linxiz) [inline portfolio/src/main/webapp/style.css:37]: ditto to remove px

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,79 @@
...
+    position: absolute;
+}
+
+/* Title Font*/
+h1{
+    color:black;
```
Comment (linxiz) [inline portfolio/src/main/webapp/style.css:43]: extra space before \*\/
extra space before \{
extra spece after \: (also applies to below lines)
Indent by 2 spaces at a time: https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,79 @@
...
+	text-decoration: none;
+	float: left;
+}
+
+li a:hover {
+	color: pink;
+}
+
+
+/*Fonts*/
```
Comment (linxiz) [inline portfolio/src/main/webapp/style.css:73]: extra spaces: "/* Fonts */"

**Pull request merged**